__tests__/CommentsTouchBox-test.js

```diff
@@ -0,0 +1,100 @@
+import 'react-native';
+import React from 'react';
+import CommentsTouchBox from '@screens/app/Components/CommentsTouchBox/CommentsTouchBox';
+import renderer from 'react-test-renderer';
+import {shallow} from 'enzyme';
+import checkPropTypes from 'check-prop-types';
+var assert = require('assert');
+
+test('like button snapshot test', () => {
+  const treeWithCommentLengthZero = renderer
+    .create(<CommentsTouchBox commentsLength={0} />)
+    .toJSON();
+  expect(treeWithCommentLengthZero).toMatchSnapshot();
+  const treeWithCommentLengthOne = renderer
+    .create(<CommentsTouchBox commentsLength={1} />)
+    .toJSON();
+  expect(treeWithCommentLengthOne).toMatchSnapshot();
+  const treeWithCommentLengthGreaterThanOne = renderer
+    .create(<CommentsTouchBox commentsLength={2} />)
+    .toJSON();
+  expect(treeWithCommentLengthGreaterThanOne).toMatchSnapshot();
+});
+
+function testPropTypesWithoutErrors(commentsLength) {
+  it(
+    'testing commentsLength values (No errors) where commentsLength is ' +
+      commentsLength,
+    () => {
+      const wrapper = shallow(
+        <CommentsTouchBox commentsLength={commentsLength} />,
+      );
+      const instance = wrapper.instance();
+      const commentsLengthProp = instance.props.commentsLength;
+      expect(commentsLength).toEqual(commentsLengthProp);
+    },
+  );
+}
+
+function testPropTypesWithErrors(commentLength, errorMsg, testDescription) {
```

Comment:
I've noticed this function a lot and began to attempt generalizing it on a page ( however we removed that component so I never fully was able to try it out before deleting) but maybe pass in the Component, full props (pre-condition that all other props except the one we want to violate meets the requirements) and then test descriptions?